### PR TITLE
docs: added celebrate 200 contributors blog

### PR DESCRIPTION
**Pull request by Serendipity96**
Changes:

docs: added celebrate 200 contributors blog

### CODE REVIEW

File: website/blog/2021-07-06-celebrate-200-contributors.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+---
+> [@Serendipity96](https://github.com/Serendipity96), Apache APISIX contributor from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+
+<!--truncate-->
+Apache APISIX has over 200 contributors in GitHub main repo!
+
+Before entering the Apache incubator, Apache APISIX had only more than 20 contributors. Today, there are more than 200 contributors to the main Apache APISIX library. 200 contributors have submitted a total of 2386 PRs, making it the most active open source gateway project in the world.
```
Comment (moonming) [inline website/blog/2021-07-06-celebrate-200-contributors.md]: `library` => `repo`

**Comment by juzhiyuan** [comment]
cc @iamayushdas to have a review

File: website/blog/2021-07-06-celebrate-200-contributors.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+>
+
+<!--truncate-->
+Apache APISIX has over 200 contributors in GitHub main repo!
+
+Before entering the Apache incubator, Apache APISIX had only more than 20 contributors. Today, there are more than 200 contributors to the main Apache APISIX repo. 200 contributors have submitted a total of 2386 PRs, making it the most active open source gateway project in the world.
+
+In August 2019, Apache APISIX released the first version 0.6.0. Version 1.0 was released in January 2020, which was the first production version of Apache APISIX. We maintain a fast and stable pace , release a new version every month. Such rapid iteration is inseparable from the support of every contributor in the community. Thanks to the community contributors for your contributions in all aspects of code, documentation, and operations!
+
+![2021-07-06-1](../static/img/blog_img/2021-07-06-1.jpg)
```
Comment (iamayushdas) [inline website/blog/2021-07-06-celebrate-200-contributors.md]: Better to use GitHub cdn image,

**Review by iamayushdas — APPROVED** [review]

File: website/blog/2021-07-06-celebrate-200-contributors.md (hunk 2 above)
Comment (Serendipity96) [reply website/blog/2021-07-06-celebrate-200-contributors.md]: ok

File: website/blog/2021-07-06-celebrate-200-contributors.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+---
+> [@Serendipity96](https://github.com/Serendipity96), Apache APISIX contributor from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+
+<!--truncate-->
+Apache APISIX has over 200 contributors in GitHub main repo!
+
+Before entering the Apache incubator, Apache APISIX had only more than 20 contributors. Today, there are more than 200 contributors to the main Apache APISIX repo. 200 contributors have submitted a total of 2386 PRs, making it the most active open source gateway project in the world.
+
+In August 2019, Apache APISIX released the first version 0.6.0. Version 1.0 was released in January 2020, which was the first production version of Apache APISIX. We maintain a fast and stable pace , release a new version every month. Such rapid iteration is inseparable from the support of every contributor in the community. Thanks to the community contributors for your contributions in all aspects of code, documentation, and operations!
```
Comment (Yiyiyimu) [inline website/blog/2021-07-06-celebrate-200-contributors.md]:
```suggestion
In August 2019, Apache APISIX released the first version 0.6.0. Version 1.0 was released in January 2020, which was the first production version of Apache APISIX. We maintain a fast and stable pace, releasing a new version every month. Such rapid iteration is inseparable from the support of every contributor in the community. Thanks to the community contributors for your contributions in all aspects of code, documentation, and maintenance!
```

File: website/blog/2021-07-06-celebrate-200-contributors.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+![poster](https://user-images.githubusercontent.com/23514812/124587288-096e2a80-de8a-11eb-94b3-95b5932c0093.jpg)
```
Comment (Yiyiyimu) [inline website/blog/2021-07-06-celebrate-200-contributors.md:17]: I'm not sure if photos stored in this way would expire somehow. Do we need to store it in the repo @juzhiyuan

File: website/blog/2021-07-06-celebrate-200-contributors.md (hunk 2 above)
Comment (Yiyiyimu) [reply website/blog/2021-07-06-celebrate-200-contributors.md]: @iamayushdas will the cdn image be expired somehow?

Comment (iamayushdas) [reply website/blog/2021-07-06-celebrate-200-contributors.md]: No it doesn't

**Review by tokers — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**